Lib/test/test_sax.py

```diff
@@ -1311,6 +1312,39 @@ def test_nsattrs_wattr(self):
         self.assertEqual(attrs.getQNameByName((ns_uri, "attr")), "ns:attr")
 
 
+# ===========================================================================
+#
+#   Sax parser property tests
+#
+# ===========================================================================
+
+class PropertyContentHandler(ContentHandler):
+    def __init__(self, test_harness, reader, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.test_harness = test_harness
+        self.reader = reader
+
+    def startElement(self, name, attr):  # @UnusedVariable
```

Comment:
Please remove the `# @UnusedVariable` comment, we don't use these in this codebase.